src/components/Common/RegionSelect.tsx

```
@@ -0,0 +1,292 @@
+
+      // 2depth만 선택한 경우 - 3depth 목록 로드
+      const isValidRegionData = REGION_DATA[depth1];
+      if (isValidRegionData && isValidRegionData[region]) {
+        setRegion3DepthData(['전체', ...isValidRegionData[region]]);
+      } else {
+        setRegion3DepthData([]);
+      }
+    },
+    [
+      currentSelectedRegion,
+      internalSelectedRegions.depth1.length,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)